doc/Language/unicode_ascii.pod6

```diff
@@ -53,7 +53,7 @@ line), or C<Zp> (Separator, paragraph) property.
 This list contains the single codepoints [and their ASCII
 equivalents] that have a special meaning in Raku.
 
-X<|«>X<|»>X<|×>X<|÷>X<|≤>X<|≥>X<|≠>X<|−>X<|∘>X<|≅>X<|π>X<|τ>X<|𝑒>X<|∞>X<|…>X<|‘>X<|’>X<|‚>X<|“>X<|”>X<|„>X<|｢>X<|｣>X<|⁺>X<|⁻>X<|¯>X<|⁰>X<|¹>X<|²>X<|³>X<|⁴>X<|⁵>X<|⁶>X<|⁷>X<|⁸>X<|⁹>X<|∈>X<|∉>X<|∋>X<|∌>X<|⊆>X<|⊈>X<|⊂>X<|⊄>X<|⊇>X<|⊉>X<|⊃>X<|⊅>X<|≼>X<|≽>X<|∪>X<|∩>X<|∖>X<|⊖>X<|⊍>X<|⊎>
+X<|«>X<|»>X<|×>X<|÷>X<|≤>X<|≥>X<|≠>X<|−>X<|∘>X<|≅>X<|π>X<|τ>X<|𝑒>X<|∞>X<|…>X<|‘>X<|’>X<|‚>X<|“>X<|”>X<|„>X<|｢>X<|｣>X<|⁺>X<|⁻>X<|¯>X<|⁰>X<|¹>X<|²>X<|³>X<|⁴>X<|⁵>X<|⁶>X<|⁷>X<|⁸>X<|⁹>X<|∈>X<|∉>X<|∋>X<|∌>X<|≡>X<|≢>X<|⊆>X<|⊈>X<|⊂>X<|⊄>X<|⊇>X<|⊉>X<|⊃>X<|⊅>X<|≼>X<|≽>X<|∪>X<|∩>X<|∖>X<|⊖>X<|⊍>X<|⊎>
```

Comment:
Is the equivalent form indexed anywhere?

Reply:
In the second commit I added the ASCII form of the ≡ operator in the heading; I hope that's enough to index it.
Anyway I grepped through the documentation, but I didn't find any other place where the ASCII operators are indexed.

Reply:
Hadn't thought about that... An oversight, no doubt.